_data/jobs.yml

```
@@ -1,3 +1,5 @@
+- {expires: 2021-09-01, location: 'West Lafayette, IN', name: 'Research Software Engineer',
```

It might be good to add the university along with the location!
```suggestion
- {expires: 2021-09-01, location: 'Purdue University, West Lafayette, IN', name: 'Research Software Engineer',
```